### PR TITLE
Use pkgCommand as root command in toit.pkg.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: tools/tpkg/main.go [hunk 1]
```diff
@@ -66,6 +83,10 @@ func main() {
 			fmt.Fprintln(os.Stderr, e)
 		}
 	}
-	rootCmd.AddCommand(pkgCmd)
-	rootCmd.Execute()
+	deprecatedPkg := copyAsDeprecated(pkgCmd)
+
+	pkgCmd.Use = "toit.pkg"
+	pkgCmd.Short = "The toit package manager"
```
Comment (kasperl) [inline tools/tpkg/main.go:89]: toit -> Toit

File: tools/tpkg/main.go [hunk 2]
```diff
@@ -22,27 +22,44 @@ import (
...
-)
-
 func getTrimmedEnv(key string) string {
 	return strings.TrimSpace(os.Getenv(key))
 }
 
 // Should be set with '-X' flag when linking.
 var sdkVersion = ""
 
+func copyAsDeprecated(cmd *cobra.Command) *cobra.Command {
```
Comment (kasperl) [inline tools/tpkg/main.go:39]: Does this mean anything for 'jag pkg'?

Comment (floitsch) [reply tools/tpkg/main.go:39]: No.
Jag uses the tpkg library the same way we do here.
Only difference is that Jag uses the pkgCommand as a subcommand and not as root command.

File: tools/tpkg/main.go (hunk 1 above)
Comment (floitsch) [reply tools/tpkg/main.go]: done.

**Review by jstroem — APPROVED** [review]

**Pull request merged**